onchain/rollups/contracts/dapp/Application.sol

```diff
@@ -123,7 +125,7 @@ contract Application is
         // reverts if proof isn't valid
         proof.validity.validateVoucher(destination, payload, epochHash);
 
-        uint256 outputIndexWithinInput = proof.validity.outputIndexWithinInput;
+        uint64 outputIndexWithinInput = proof.validity.outputIndexWithinInput;
```

Comment:
we could also keep this as `uint256` and call `toUint64()` when emitting `VoucherExecuted`, like you did for `inputIndex`

Reply:
We could, but it would be super weird to cast a `uint64` to a `uint256` then call `toUint64` on it.